amgut/lib/data_access/ag_data_access.py

```diff
@@ -333,7 +344,9 @@ def deleteAGParticipantSurvey(self, ag_login_id, participant_name):
             sql = """INSERT INTO ag.consent_revoked
                      (ag_login_id,participant_name, participant_email)
                      VALUES (%s, %s, %s)"""
-            TRN.add(sql, [ag_login_id, participant_name, participant_email])
+            for participant_email in list(participant_emails):
```

Comment:
I don't think the call to `list` is needed (you can iterate over a set). However, this can be changed to:
```python
sql_args = [[ag_login_id, participant_name, pemail] for pemail in participant_emails]
TRN.add(sql, sql_args, many=True)
```

And add at the end of the function:
```python
TRN.execute()
```

So we make sure that whatever is listed here is actually executed before returning this functions (but not necessarily committed)